File: Iniciante/1051 - Imposto de Renda/Python/1051.py
def main():
    salary = float(input())
    value_income_tax = calculate_income_tax(salary)
    print_result(value_income_tax)


def calculate_income_tax(salary):
    tax = 0.0

    if salary > 4500.00:
        tax += (salary - 4500.00) * 0.28
        salary = 4500.00

    if salary > 3000.00:
        tax += (salary - 3000.00) * 0.18
        salary = 3000.00

    if salary > 2000.00:
        tax += (salary - 2000.00) * 0.08

    return tax


def print_result(value_income_tax):
    if value_income_tax == 0.0:
        print("Isento")
    else:
        print(f"R$ {value_income_tax:.2f}")


if __name__ == "__main__":
    main()

File: Iniciante/1051 - Imposto de Renda/JavaScript/1051.js
function main() {
    const input = require("fs").readFileSync("/dev/stdin", "utf8");
    const salary = parseFloat(input);
    const valueIncomeTax = calculateIncomeTax(salary);
    print(valueIncomeTax);
}

function calculateIncomeTax(salary) {
    let tax = 0.0;

    if (salary > 4500.00) {
        tax += (salary - 4500.00) * 0.28;
        salary = 4500.00;
    }

    if (salary > 3000.00) {
        tax += (salary - 3000.00) * 0.18;
        salary = 3000.00;
    }

    if (salary > 2000.00) {
        tax += (salary - 2000.00) * 0.08;
    }

    return tax;
}

function print(valueIncomeTax) {
    if (valueIncomeTax === 0.0) {
        console.log("Isento");
    } else {
        console.log(`R$ ${valueIncomeTax.toFixed(2)}`);
    }
}

main();
